specification/eventgrid/data-plane/Microsoft.EventGrid/stable/2024-06-01/EventGrid.json

```diff
@@ -0,0 +1,885 @@
+        "parameters": [
+          {
+            "$ref": "#/parameters/Azure.Core.Foundations.ApiVersionParameter"
+          },
+          {
+            "name": "topicName",
+            "in": "path",
+            "description": "Topic Name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "eventSubscriptionName",
+            "in": "path",
+            "description": "Event Subscription Name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "acknowledgeOptions",
+            "in": "body",
+            "description": "AcknowledgeOptions.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AcknowledgeOptions"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/AcknowledgeResult"
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/Azure.Core.Foundations.ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "String error code indicating what went wrong."
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Acknowledge Cloud Event": {
+            "$ref": "./examples/ack.json"
+          }
+        }
+      }
+    },
+    "/topics/{topicName}/eventsubscriptions/{eventSubscriptionName}:release": {
+      "post": {
+        "operationId": "ReleaseCloudEvents",
+        "description": "Release batch of Cloud Events. The server responds with an HTTP 200 status code if the request is successfully accepted. The response body will include the set of successfully released lockTokens, along with other failed lockTokens with their corresponding error information.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Azure.Core.Foundations.ApiVersionParameter"
+          },
+          {
+            "name": "topicName",
+            "in": "path",
+            "description": "Topic Name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "eventSubscriptionName",
+            "in": "path",
+            "description": "Event Subscription Name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "releaseDelayInSeconds",
+            "in": "query",
+            "description": "Release cloud events with the specified delay in seconds.",
+            "required": false,
+            "type": "number",
```

Comment:
It is odd to have an `enum` of type `number` and particularly odd when all the values for the enum are actually integers. In Azure we generally want enums to be `string`, and particularly when `x-ms-enum` has `modelAsString: true`.

I think it would probably be best to make this a string.
```suggestion
            "type": "string",
```

This will require wrapping all the enum values in quotes.

Reply:
Changed the enum values from number to string.